ci/deploy.sh

```diff
@@ -61,19 +70,33 @@ FIRST_FILE=${CLASS_FILES%%,*}
 cp -f "$FIRST_FILE" "$FPATH.jar"
 
 ###### Build the deploy command ######
-MVN="mvn -Dmaven.wagon.http.retryHandler.count=3 -DretryFailedDeploymentCount=3 -B"
 if [ "$SIGN_FILE" == true ]; then
-    DEPLOY_CMD="$MVN -B gpg:sign-and-deploy-file -s ci/settings.xml \
-      -Dsources=$FPATH-sources.jar -Djavadoc=$FPATH-javadoc.jar -Dgpg.passphrase=$GPG_PASSPHRASE"
+    case $SIGN_TOOL in
```

Comment:
nit: can we also put a default `gpg` for SIGN_TOOL?

Reply:
Good catch! updated